### ISSUE TITLE
Milestone 2 feedback

### ISSUE THREAD

**Issue by kandarpksk** [dingqixin/chicas#22]

**Comment by Elkhiffa** [comment]
Team 10

Yuxiao Fang:
The first 2 storyboards are too wordy. The trip planning idea is kind of vague, seems like what the app does is only about lining up spaces the user picked, though the storyboard mentioned optimizing route, how is the route optimized by what standards in which aspects are not mentioned. 

Miguel Vargas:
I feel like storyboards 1 & 7 offer the best app idea as the others seem like an app could be a bit unrealistic or its target audience is too specific that not many users will use it. 1 & 7 offer fun or useful ideas that can augment someone's experience while running or traveling. 

Chen Liu:
I looked at the storyboards No.3 and 4. For the storyboard 3, I really like this idea as it actually coincides with one of my ideas. The lost-and-found app could be extremely useful and extensible. However, the functionalities of the other one may not be persuasive enough to let the users use them. User may not have the motivation to contribute their stories, especially those private stories to the public.

**Comment by nonobitata** [comment]
Team 11: 
Overall:
- Text heavy.
- Some of the storyboards need to be more about the situation. This is with the consideration that a few of the storyboards just explained the app rather than the situation it would be needed in.
- Dont do on sticky note


Storyboard 2:
- The example is not practical, asking someone to hand in your homework is not ok.
- It should also include your location so people can come and pick things up for you.

Storyboard 4:
- Marking where you have been may not be relevant to finding your lost item. People can still mark on the map (you just need to remember where you went to and it cost a lot less than keeping track your activities everyday)

Storyboard 8: 
- Not really relevant to geo/map, is it a reminder?

**Comment by akshaymahajans** [comment]
Team C3 - TBD - Akshay Mahajan

Storyboard 7 seems like a simple, useful app. It has been done before, and could be pretty easily implemented. Perhaps adding a less-used feature could make your app more novel and possibly more useful.

Storyboard 8 is interesting! If you could parse key words about particular places close by, and assign a location to each place that matches key words/phrases, that would be a really cool app idea.

**Comment by optimmyze** [comment]
Team 12/C3 - TBD
Review 5 & 6
5: a pretty cool travelling application. It would be good for user to user and be nice with ratings. really smart. But itll take to to startup all over the world.
6: this is a neat idea. very challenging to friends. Easy to push each other.

**Comment by princevietle** [comment]
Team C3 - TBD - Prince Le

Storyboard 3:
I really like the idea of a lost and found map, as most people don’t tend to remember exactly where they have been throughout the day.  A suggestion is to also include how long they were at a location because knowing that they have been at location A for an hour and location B for only 5 minutes may be useful to someone who might need to find their item as soon as possible.  Also, another suggestion would be to have some quick and easy button to tell the app that “I have everything I need” at a certain time interval, so if they were to lose an item after this interval, they can figure out which locations they need to search in order to find their item.

Storyboard 4:
I like the idea of seeing the stories that occur at a certain place or building.  My suggestion is to have some way to display all the possible stories possibly with a upvote or downvote system, since you would want to see the best stories of each building.  Also, would the user have to be in the proximity in order to add a story to a location?  And would only specific landmarks be able to add stories?  There should be a very simple and easy way to display the possible stories to the user without cluttering or crowding the user’s screen.

**Comment by siru-a** [comment]
Team C3 - TBD 
Storyboard 1: I like the idea about trip planning, because I often have trouble get all visiting places organized nicely. I think it'll be cool if the app can show users the cheapest way/ fastest way for going from one place to the next visting place. Also, it can be helpful if the app can gather many users that are travelling to the same place.

Storyboard 2:
The app does a good job at distributing task, but I think the time required for people to move from one place to another might be a problem. For example, say I need to hand in homework in 10 minutes, but the person who can help me is 5 minute away, so even though she can help me hand in the homework, she won't arrive at destination on time..So the app should consider situations like this.

**Comment by koahmad** [comment]
Team C4 (Team 7meer):
We really liked your lost and found idea. It's a unique idea that we haven't seen before. Something you can add to the idea is adding what you have on you. For example, if you are in Geisel you have your keys. But at the gym, you usually don't bring your keys. 

We also liked your exploring map idea. It would work very well for freshman or newcomers to campus in order to get oriented to the culture of the school. One thing to think about is how you will curate the stories that will be shown, so that you do not have any fake or inappropriate information.
